src/content/templates/template books/Games/Games.html

```diff
@@ -277,60 +277,32 @@
             </div>
         </div>
 
-        <!-- Hiding the Drag Letters to Targets game for now. It isn't ready for the FAL channel. BL-14696 -->
-        <!-- <div
-            class="bloom-page customPage bloom-ignoreForReaderStats bloom-interactive-page no-margin-page numberedPage side-right Device16x9Landscape bloom-monolingual draggables-need-shuffling"
-            id="c8f4834c-1efd-4864-bc2c-8538ef26bb5b"
-            data-feature="game"
+        <div
+            class="bloom-page targets-semi-transparent customPage bloom-ignoreForReaderStats bloom-interactive-page no-margin-page numberedPage side-right Device16x9Landscape bloom-monolingual draggables-need-shuffling"
```

Comment:
Let's remove redundant (and especially contradictory) classes like Device16x9Landscape and side-left/right. They might confuse future human editors. (In some places Device16x9Landscape is on pages marked data-initial-page-orientation="portrait"!)